### ISSUE TITLE
Popup messageboxes/windows are annoying

### ISSUE THREAD

**Issue by AlexStrobel** [keybase/keybase-issues#2532]
I have installed the keybase io app, but did not enter the passphrase. Now after some time the app pops up an input box and wants me to input my passphrase.
This is annoying, as it 1st) interrupts my workflow and 2nd) I can not close the window with ESC key.

This behavior is also annoying when updates are available or one installed the latest update and there is a restart of the PC required. In that case it was even so, that the messagebox came up just before I hit Enter resulting in a restart of my computer. Luckily I had no unsaved changes.

**Comment by AlexStrobel** [comment]
If I were able to close/quit the app, it would help me very much.

**Comment by zanderz** [comment]
If you click the icon in the tray, there should be a small window with a menu icon in the upper left, which offers a "quit" option.

Reboots are tricky but only an issue when drivers are updated, which is seldom.

**Comment by gabriel** [comment]
Yeah that is a little annoying. We're working on some fixes to make this password dialog not come up after reboots and restarts. Hopefully soon-ish.

**Comment by zQueal** [comment]
I've also opened a similar ticket #2480. @maxtaco has indicated that a fix is in the works ASAP.

**Comment by AlexStrobel** [comment]
@zanderz 
I simply did overlook this small symbol... Thanks for this hint :)

Thanks everyone for your responses.
Besides the (not anymore so...) annoying popup the "problem" with the popup for a restart still exists. Would be great if it could be fixed somewhen. :)
